examples/lujain_work_area/Contact_Book/contact_records.py

```diff
@@ -0,0 +1,47 @@
+# contact_records.py
+
+import csv
+import os
+
+class ContactRecords:
+    def __init__(self, csv_file):
+        self.csv_file = csv_file
+        self.fieldnames = ['Name', 'Phone', 'Email', 'Address']
+        self.contacts = self.load_contacts()
+
+    def load_contacts(self):
+        if not os.path.exists(self.csv_file):
+            return []
+        with open(self.csv_file, mode='r') as file:
+            reader = csv.DictReader(file)
+            return [row for row in reader]
+
+    def save_contacts(self):
```

Comment:
ContactRecord should not be responsible for dealing with filing. In real life we write the record to a ContactBook so I believe ContactBook should deal with storing and loading.